### ISSUE TITLE
Will not operate standalone

### ISSUE THREAD

**Issue by elaske** [vdmeer/svg2vector#2]
I tried downloading the *.jar file from maven repos (1.0.3). The readme doesn't include anything on how to use it, so I found `Tool` which includes a `main()` that seems to take CLI parameters. I tried to call it from the command line, but it doesn't load the class correctly:

```
$ java -cp svg2vector-1.0.3.jar de.vandermeer.svg2vector.Tool --help
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/commons/cli/ParseException
    at de.vandermeer.svg2vector.Tool.<init>(Tool.java:68)
    at de.vandermeer.svg2vector.Tool.main(Tool.java:59)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.cli.ParseException
    at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
    at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
    ... 2 more
```

**Comment by elaske** [comment]
It looks like the available jar on the maven repositories does not include dependencies. Without the jar, `svg2vector-1.0.3-jar-with-dependencies.jar`, it will not run correctly. Luckily, it's relatively straightforward to call it with the dependencies:

```
$ java -jar target/svg2vector-1.0.3-jar-with-dependencies.jar 
svg2vector: target required, see -h for details
```

I think this example call should also be in the README.

**Comment by vdmeer** [comment]
Yes, there was no explanation on how to actually run the tool. Sorry for that.

Fixed this now, in version 1.1.0 (just build and pushed). The README has a few sections on how to use/run the application with or without building it.

I hope this solves this issue!

Thanks for your comment!
/sven
